spec/factories.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
+  factory :piece do
+    
```

Comment:
Yes - need to add to factorygirl - will do that and will add (create) one for game too.